test/integration/alertsMenuView_integrationTests.js

```diff
@@ -128,25 +151,32 @@ describe('alertsMenuView.js spec', function() {
     });
     describe('view tests', function() {
         it('renders alerts accordingly', function() {
+            expect($('.alerts-all').html()).to.equal('');
             this.server.respond();
-            expect($('.alerts-all').html()).to.equal('<li><div class="msg-block"><span class="msg">Alert: \'service status DOWN\' triggered at 2016-03-10 00:28:00+00:00</span></div></li>');
+            expect($('.alerts-all').html()).to.equal('<li><div class="msg-block"><span class="msg">Alert: \'service status DOWN\' triggered at ' + formatDate(now_date) + ' 00:01:00+00:00</span></div></li><li><div class="msg-block"><span class="msg">Alert: \'service status DOWN\' triggered at ' + formatDate(old_date) + ' 00:01:00+00:00</span></div></li>');
         });
         it('renders recent alerts accordingly', function() {
             expect($('.alerts-recent').html()).to.equal('');
             this.server.respond();
-            // greater than a day
-            expect($('.alerts-recent').html()).to.equal('');
+            expect($('.alerts-recent').html()).to.equal('<li><div class="msg-block"><span class="msg">Alert: \'service status DOWN\' triggered at ' + formatDate(now_date) + ' 00:01:00+00:00</span></div></li>');
         });
         it('sets an empty model to register changes against', function() {
             this.testView.setModel();
             expect(this.testView.model.get('alerts')).to.deep.equal([]);
             this.server.respond();
             expect(this.testView.model.get('alerts')).to.not.deep.equal([]);
         });
-        it('highlights the alert icon when the model changes', function() {
+        it('highlights the alert icon when the model changes and includes recent alerts', function() {
             expect($(this.testView.el).hasClass('alert-active')).to.equal(false);
             this.server.respond();
             expect($(this.testView.el).hasClass('alert-active')).to.equal(true);
+            serverResult.results = [alert_old];
```

Comment:
Each test should not depend on side effects from a previous test. It looks like the declaration on the line 173 is provided so that the next test will pass. Instead, move the declaration from line 173 into the test that it is affecting, on line 176.